### PR TITLE
Adding Draft Endpoint

### PR DESCRIPTION
**Pull request by bradleyhurley**

### CODE REVIEW

File: PyBall/models/draft/person.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+                 birthDate=None, birthCountry=None, primaryPosition=None, batSide=None,
+                 pitchHand=None, nameSlug=None):
+        self.id = id
+        self.fullName = fullName
+        self.link = link
+        self.firstName = firstName
+        self.lastName = lastName
+        self.birthDate = birthDate
+        self.birthCountry = birthCountry
+        self.primaryPosition = PrimaryPostion(**primaryPosition)
```
Comment (a-hacker) [inline PyBall/models/draft/person.py:17]: We should consider how to deal with nested objects like these that will throw errors on bad data (like with the Dominican Summer League). However the `or {}` approach adds a lot of repetition and room for error if and when we forget about it. It's not something to tackle in this PR but just something to keep in mind.

File: PyBall/models/draft/prospect.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+from PyBall.models.draft.home import Home
+from PyBall.models.draft.school import School
+from PyBall.models.draft.person import Person
+
+
+class Prospect:
```
Comment (a-hacker) [inline PyBall/models/draft/prospect.py:6]: Prospect and Pick seem to be identical. Should they be combined?

File: PyBall/models/draft/round.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+from PyBall.models.draft.pick import Pick
+
+
+class Round:
+    def __init__(self, roundNumber=None, picks=None):
+        self.roundNumber = roundNumber
+        self.picks = []
```
Comment (a-hacker) [inline PyBall/models/draft/round.py]: `self.picks = [Pick(**pick) for pick in picks]`?

File: PyBall/models/draft/draft.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+from PyBall.models.draft.round import Round
+
+
+class Draft:
+    def __init__(self, rounds=None, draftYear=None):
+        self.draftYear = draftYear
+        self.rounds = []
```
Comment (a-hacker) [inline PyBall/models/draft/draft.py]: `self.rounds = [Round(**round) for round in rounds]`

**Review by a-hacker — DISMISSED** [review]
I don't see any issues. Added a few comments about things to consider.

File: PyBall/models/draft/prospect.py (hunk 2 above)
Comment (bradleyhurley) [reply PyBall/models/draft/prospect.py:6]: Im not against it, but Im not sure what to name name the more generalized class.

**Review by a-hacker — APPROVED** [review]

**Pull request merged**